example/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: Demonstrates how to use the freerasp plugin.
 publish_to: 'none'
 
 environment:
-  sdk: ">=2.12.0 <4.0.0"
+  sdk: ">=3.0.0 <4.0.0"
```

Comment:
I think we need even higher one
`Because freerasp depends on pigeon >=19.0.0 which requires SDK version >=3.2.0 <4.0.0, version solving failed.`

Reply:
Also version of gradle should be upgraded in settings.gradle, I wasn't able to build the project with current settings

Reply:
You shouldn't need that in example. `pigeon` is dev dependency in Talsec SDK itself, not example. 

In the Talsec SDK, I didn't even raised version, but I'll upgrade both to some newer version. `2.18.0` is pretty old anyway.